e2e/suite1/add-language.test.ts

```diff
@@ -24,9 +24,7 @@ describe('Add language', () => {
     await expect(page).toClick('span', { text: 'Show in cards' });
     await expect(page).toClick('span', { text: 'Save' });
     await expect(page).toClick('span', { text: 'Languages' });
-    await expect(page).toClick(
-      'ul.list-group.document-types > li:nth-child(1) > div > button.template-remove'
-    );
+    await expect(page).toClick('button', { text: 'Add language' });
```

Comment:
Generally this test is difficult to understand. Why does it matter if it's changing template values or not? 
If time permits and it's something useful, this test should be divided in more descriptive it's. And perhaps remove altogether the process of editing a template if it's inconsequential.

Reply:
Just for the record, @Zasa-san and I updated the test to make it more related with languages